bundles/org.openhab.binding.nikobus/README.md

```diff
@@ -37,48 +35,21 @@ Bridge enables communication with other Nikobus components:
 
 ## Discovery
 
-In order to configure items in openHAB, one first needs to know the addresses of switch-modules, dimmer-modules, rollershutter-modules and push-buttons.
-
-Binding's root logger is `org.openhab.binding.nikobus`.  Enable `DEBUG` logging to discover addresses that will be logged.  In openHAB 2, this can be performed from the console at the `openhab>` prompt:
-
-```
-log:set DEBUG org.openhab.binding.nikobus
-log:tail
-```
-
-Press Control-C to stop monitoring the log and return to the prompt.
-
-Please see [here](https://www.openhab.org/docs/administration/logging.html) for more details on the subject.
-
-Once configured, each push button press should produce a log entry as:
-
-```
-... Received command '#N40F9CA' ...
-```
-
-The six characters following `#N` are the push button's address, `40F9CA` in the example above.
-
-To find out the address of module(s), press and hold the yellow "mode" button on the module until it beeps. This will trigger the module to send out its identification on the bus, which should then be logged in openHAB.  In the openHAB's log, one should see an entry similar to:
-
-```
-... Received command '$18C96400100167FF78607` ...
-```
-
-The four characters following `$18` are the module's address. In the example above, that's `C964`.
+The binding does not support any automatic discovery of Things.
```

Comment:
@crnjan, if I understand it correct, there is no discovery mechanism in place, so this section should be empty.

I wonder though if it wouldn't possible to implement a discovery, when there is a way to let the devices announce themselves ("yellow button until it beeps")?